packages/at_client/lib/src/service/sync_service_impl.dart

```diff
@@ -422,7 +431,8 @@ class SyncServiceImpl implements SyncService, AtSignChangeListener {
       }
```

Comment:
LGTM. While we're changing the code here, can you also log the stack trace when we encounter exceptions or errors please